src/Goteo/Controller/Admin/ImpactDataAdminController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+
+namespace Goteo\Controller\Admin;
+
+use Goteo\Application\Exception\ModelNotFoundException;
+use Goteo\Application\Message;
+use Goteo\Library\Text;
+use Goteo\Library\Forms\Model\ImpactDataForm;
+use Goteo\Model\ImpactData;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Route;
+
+class ImpactDataAdminController extends AbstractAdminController
+{
+    protected static $icon = '<i class="fa fa-2x fa-table"></i>';
```

Comment:
Why do we need HTML code in a PHP controller?

Reply:
Since @javicarrillo and @microstudi developed the new modules, they have this protected $icon that is used to show, normally, a font-awesome icon and it has been always added through that html code inside the variable. We could change it.

Reply:
Cool, we could later rewrite this